web/libs/ui/src/lib/select/select.tsx

```diff
@@ -77,6 +78,7 @@ export const Select = forwardRef(
       selectedValueRenderer,
       selectFirstIfEmpty,
       renderSelected,
+      useInfiniteScroll = false,
```

Comment:
I would rename this to:


```suggestion
      isVirtualList = false,
```